src/styles/blocks/product.scss

```diff
@@ -0,0 +1,167 @@
+.product {
+  &--catalog {
+    padding-block: 50px;
+    padding-inline: 40px;
+    display: grid;
+    grid-template-rows: 2;
+    grid-template-columns: repeat(1, 200px);
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 200px);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat(3, 200px);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding-left: 16px; /*для сдачи теста*/
+    display: flex;
+    box-sizing: border-box;
+    width: 200px;
+    height: min-content;
+    flex-direction: column;
+    text-align: center;
+    border: 1px solid $whiteColor;
+    border-radius: 5px;
+    transition: 300ms;
```

Comment:
transition-property

Reply:
<img width="629" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/e21c7f2e-ebef-47a3-a928-e355d53e2f5e">
better to add transition property